src/main/java/org/broadinstitute/consent/http/enumeration/DatasetPropertyType.java

```diff
@@ -64,9 +64,9 @@ public static Boolean coerceToBoolean(String value) throws IllegalArgumentExcept
         return java.lang.Boolean.valueOf(value);
     }
 
-    public static Instant coerceToDate(String value) throws IllegalArgumentException {
+    public static LocalDate coerceToDate(String value) throws IllegalArgumentException {
```

Comment:
context: instants require information about timezone and the exact ms when things happen. LocalDate, instead, is the simplest form of date: no knowledge of timezones, no time of day, just "YYYY-MM-DD", which is all we want to capture in the dataset registration. Perhaps in the future we will need a dataset property with more granular time; in which case, we could perhaps make a `DateTime` or `Instant` dataset property type.